eve/main.yml

```diff
@@ -1641,6 +1643,14 @@ stages:
           name: Set OS property to rhel-8
           property: os
           value: "rhel-8"
+      - SetProperty:
+          name: Set offline property to false
+          property: offline
+          value: false
+      - SetProperty:
+          name: Set use_proxy property to false
+          property: use_proxy
+          value: false
```

Comment:
Worth adding a comment + link to a ticket (not sure if we've created one yet) to remember why this is disabled and what we should do to re-enable it.